src/modules/CreateChannel/components/InviteUsers/utils.ts

```diff
@@ -29,7 +29,9 @@ export const createDefaultUserListQuery = (
   { sdk, userFilledApplicationUserListQuery }: CreateDefaultUserListQueryType,
 ): ApplicationUserListQuery => {
   if (sdk?.createApplicationUserListQuery) {
-    const params = sdk?.createApplicationUserListQuery();
+    const params = sdk?.createApplicationUserListQuery({
+      userIdsFilter: ['bot_test_0a780'],
+    });
```

Comment:
do we have to remove this line?

Reply:
ah yes thanks for the reminder. its for testing only and need to remove before merging.